TwoPageEBookReader/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.ebook_reader_sample">
```

Comment:
I've seen thar you guys are using `com.example.foo.bar` as package for all of your projects, maybe you can consider to use instead `com.microsoft.device.wm_samples.foo.bar` or something like that? cc / @conceptdev

Reply:
done